tests/js/spec/views/dashboardsV2/detail.spec.jsx

```diff
@@ -289,12 +289,20 @@ describe('Dashboards > Detail', function () {
         .find('IconClick[data-test-id="widget-delete"]')
         .simulate('click');
 
+      // Confirm
+      const FirstModal = await mountGlobalModal();
+      FirstModal.find('Button').last().simulate('click');
+
       wrapper
         .find('WidgetCard')
         .at(1)
         .find('IconClick[data-test-id="widget-delete"]')
         .simulate('click');
 
+      // Confirm
+      const SecondModal = await mountGlobalModal();
```

Comment:
You shouldn't actually have to re-mount the modal container, which is what `mountGlobalModal` does. you could just mount `modalContainer` in the beginning and `find` on it twice

Reply:
Discussed this further in slack and wasn't able to get this to work, going with 2x mountGlobalModal for now since we’ll convert this test away from enzyme at some point anyawys